src/app/features/mushaf/pages/ayah-focus/ayah-focus.page.less

```
@@ -0,0 +1,113 @@
+    &--disabled {
+      opacity: 0.45;
+      pointer-events: none;
+    }
+  }
+
+  &__ayah {
+    padding: 2rem 1.5rem;
+    border: 1px solid var(--color-border, #e5e7eb);
+    border-radius: var(--radius-lg, 14px);
+    background: var(--color-surface, #fff);
+    font-family: 'Amiri', 'Scheherazade New', 'Traditional Arabic', serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Amiri" (font-family-name-quotes)

(font-family-name-quotes)